server/fleet/orbit.go

```diff
@@ -43,12 +43,6 @@ type OrbitConfigNotifications struct {
 	// enabled and the device should encrypt its disk volumes with BitLocker.
 	EnforceBitLockerEncryption bool `json:"enforce_bitlocker_encryption,omitempty"`
 
-	// EnableBitLockerPINProtectorConfig is set if Windows MDM is enabled, BitLocker encryption is
-	// enforced, and the RequireBitLockerPIN server config flag is set. If set, this will
-	// make sure that the BitLocker policy is configured correctly so that the user can configure a
-	// TPM PIN protector.
-	EnableBitLockerPINProtectorConfig bool `json:"enable_bitlocker_pin_protector_config,omitempty"`
```

Comment:
This is ok to remove since the policy config enforcement is implemented using MDM and not the Agent